sdk/cognitivelanguage/azure-ai-language-conversations/azure/ai/language/conversations/_operations/_operations.py

```diff
@@ -90,7 +90,7 @@ def analyze_conversation(self, task: JSON, *, content_type: str = "application/j
         """Analyzes the input conversation utterance.
 
         See
-        https://learn.microsoft.com/rest/api/language/conversation-analysis-runtime/analyze-conversation
+        https://learn.microsoft.com/rest/api/language/2022-10-01-preview/conversation-analysis-runtime/analyze-conversation
```

Comment:
That's where they stuck the api-version?! This is going to be a pain to keep up to date moving forward.

Reply:
I'll find some way to provide feedback to the ~Docs~ Learn team.